Dockerfile

```diff
@@ -11,23 +11,30 @@ COPY go.mod go.mod
 COPY go.sum go.sum
 
 # ensure vendoring is up-to-date
-# cache deps before building and copying source so that we don't need to 
-# re-download as much and so that source changes don't invalidate our 
+# cache deps before building and copying source so that we don't need to
+# re-download as much and so that source changes don't invalidate our
 # downloaded layer
 RUN make vendor
 
 # copy go source code
-COPY cmd/ cmd/
-COPY pkg/ pkg/
+COPY cmd/ ./cmd/
+COPY pkg/ ./pkg/
+COPY k8s/ ./k8s/
+COPY types/ ./types/
+COPY controller/ ./controller/
 
 # build the binary
-RUN make lbins
+RUN make bin
 
-# Use distroless as minimal base image to package the manager binary
-# Refer to https://github.com/GoogleContainerTools/distroless for more details
-FROM gcr.io/distroless/static:latest
+# Use debian as minimal base image to package the final binary
+FROM debian:stretch-slim
```

Comment:
we shall use distroless

Reply:
Done 
https://github.com/mayadata-io/oep-operator/pull/5/